readthedocs/projects/models.py

```diff
@@ -902,6 +903,7 @@ class ImportedFile(models.Model):
     path = models.CharField(_('Path'), max_length=255)
     md5 = models.CharField(_('MD5 checksum'), max_length=255)
     commit = models.CharField(_('Commit'), max_length=255)
+    is_html = models.BooleanField(default=False)
```

Comment:
I don't think we need this on the model, the queryset manager can just do the filter, no?

Reply:
Yeah. it can do the filtering. but I thought it would be much slow to filter in the queryset manager. I am ok to remove it.